src/Pages/Home.js

```diff
@@ -0,0 +1,132 @@
+              />
+            </Link>
+
+            <div className="video-card-content">
+              <Link
+                to={`/video/${video.id}`}
+                className="link-btn"
+                onClick={() => addToHistory(video)}
+              >
+                <Avatar alt="avatar" src={video.image} size="40px" />
+              </Link>
+
+              <div style={{ paddingLeft: "10px" }}>
+                <div className="flex">
+                  <Link
+                    to={`/video/${video.id}`}
+                    style={{ width: "230px" }}
+                    className="bold-text link-btn"
+                    onClick={() => addToHistory(video)}
+                  >
+                    {formatString(video.title)}
+                  </Link>
+                  <div className="tooltip pointer">
+                    <HiDotsVertical size="1.5rem" color="grey" />
+                    <div
+                      style={{ color: "grey" }}
+                      className="tooltiptext card-shadow light-border"
+                    >
+                      {!CheckIfVideoExistsInList(watchLater, video.id) ? (
+                        <div
+                          className="flex padding-bottom link-btn list-item-padding list-item"
+                          onClick={() => addToWatchLater(video)}
+                        >
+                          <MdWatchLater size="1.2rem" />
+
+                          <span className="padding-left">
+                            Save to Watch Later
+                          </span>
+                        </div>
+                      ) : (
+                        <div
+                          className="flex padding-bottom link-btn list-item-padding list-item"
+                          onClick={() => addToWatchLater(video)}
+                        >
+                          <MdRemoveCircle size="1.2rem" />
+
+                          <span className="padding-left">
+                            Remove from watch later
+                          </span>
+                        </div>
+                      )}
+
+                      <div
+                        className="flex padding-bottom link-btn list-item-padding list-item"
+                        onClick={() => handlePlaylist(video.id)}
+                      >
+                        <MdPlaylistAdd size="1.2rem" />
+
+                        <span className="padding-left">Save to Playlist</span>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+
+                <Link
+                  to={`/video/${video.id}`}
+                  style={{ color: "grey" }}
+                  className="link-btn"
+                  onClick={() => addToHistory(video)}
+                >
+                  {video.author}
+                </Link>
+                <br />
+                <Link
+                  to={`/video/${video.id}`}
+                  style={{ color: "grey" }}
```

Comment:
General comment: If you can move your styles to css file then please consider to move it as it will help you in keeping code clean. Please check all files.